research/manuscript-example/manuscript.tex

```diff
@@ -0,0 +1,188 @@
+\documentclass[conf]{new-aiaa}
+%\documentclass[journal]{new-aiaa} for journal papers
+\usepackage[utf8]{inputenc}
+
+\usepackage{graphicx}
+\usepackage{amsmath}
+\usepackage[version=4]{mhchem}
+\usepackage{siunitx}
+\usepackage{longtable,tabularx}
+\setlength\LTleft{0pt} 
+
+\title{Model-Based CubeSat Flight-Software Architecture using a Docs-as-Code approach}
+
+\author{Sean Marquez and Kevin Chiu}
+\affil{Old Dominion University, Norfolk, VA, 23529}
+\author{Sharan Asundi, Ph.D. \\footnote{Assistant Professor, Department of Mechanical & Aerospace Engineering, Old Dominion University}}
+\affil{Old Dominion University, Norfolk, VA, 23529}
+
+\begin{document}
+
+\maketitle
+
+\begin{abstract}
+PLACEHOLDER
+\end{abstract}
+
+\section{Nomenclature}
+
+{\renewcommand\arraystretch{1.0}
+\noindent\begin{longtable*}{@{}l @{\quad=\quad} l@{}}
+$1U$ 1-Unit \\
+$3U$ 3-Unit \\
+$BOM$  & Bill of Materials \\
+$ConOps$  & Concept of Operations \\
+$DOF$  & Distributed OSHW Framework  \\
+$m30ml$  & mach30 modeling language \\
+$MBSE$ &    model-based systems engineering  \\
+$ODU$& Old Dominion University \\
+$OML$  & Ontological Modeling Language \\
+$OSHW$ & Open Source Hardware \\
+$OWL2$ & Web Ontology Language 2 \\
+$SWRL$ & Semantic Web Rule Language \\
+$USCGA$ & United Stated Coast Guard Academy \\
+$WFF$   & Wallops Flight Facility  
+\end{longtable*}}
+
+\section{Introduction}
+\lettrine{T}{his} article presents the Mach30 modeling language (m30ml), tools, and technical approach used to facilitate the configuration management, design, specification, & implementation of the SeaLion mission architecture for the flight software.
```

Comment:
verbiage should be more focused on technical approach, i.e., filesystem-based MBSE. Mention of Mach30 language or toolchain should be in a section that describes our implementation of our approach

Reply:
i.e., models persistent on the local filesystem.

Reply:
tradestudy for background content https://github.com/ODU-CGA-CubeSat/sealion-mission-architecture/discussions/40

Reply:
most toolchains are intended for cloud-based collaboration

Reply:
desired outcomes
1. Ensure templates only contain formatting data (this includes not storing boilerplate text in templates)
2. Ensure models are the authoritative source of truth for all artifact content (e.g. artifact structure, meta-data, boilerplate, commentary, discussion, diagrams, tables, etc)
3. models should persist on the local filesystem